extensions/interactions/TextInput/directives/text-input-validation.service.ts

```diff
@@ -67,24 +67,27 @@ export class TextInputValidationService {
       });
     }
 
-    let isInt = function(n) {
-      return angular.isNumber(n) && n % 1 === 0;
+    let isInt = (n: number) => {
+      return ((typeof (n) === 'number') && (n % 1 === 0));
```

Comment:
```suggestion
      return ((typeof(n) === 'number') && (n % 1 === 0));
```

Reply:
This throws an es-lint error so we'll have to leave a space

Reply:
Then maybe just do `typeof n` without the brackets.

Reply:
Done